client/src/relatedProducts/RelatedProductsView/RelatedProductsList.jsx

```diff
@@ -9,8 +9,8 @@ const RelatedProductsList = (props) => {
     <div className='relatedProductsListContainer'>
       <h2>Related Products:</h2>
     <div className='relatedProductsList'>
-    {props.allProps.map((product, i) => {
-      return <div key={i}>
+    {props.allProps.map((product, index) => {
+      return <div key={index}>
```

Comment:
this was just added to remove the errors in client console being received for not having a key